tritium-metrics/src/test/java/com/palantir/tritium/event/metrics/TaggedMetricsServiceInvocationEventHandlerTest.java

```diff
@@ -34,9 +34,9 @@
 
     public static final class TestImplementation {
 
-        @SuppressWarnings("unused")
+        @SuppressWarnings("unused") // instrumented
         public String doFoo() {
-            return "bar";
+            return this.getClass().getSimpleName();
```

Comment:
this wasn't related to generics, but IntelliJ was prompting that this method could be static, so just wanted that warning to go away so class would be green.